### ISSUE TITLE
RuntimeError: Overflow reading light channels! could maybe suggest lowering the gain

### ISSUE THREAD

**Issue by jedgarpark** [adafruit/Adafruit_CircuitPython_TSL2591#20]
[Sample code](https://learn.adafruit.com/adafruit-tsl2591/python-circuitpython#full-example-code-2997857-15) is at 25x gain so a decent flashlight will exceed the light level pretty easily and crash the program with the error `https://learn.adafruit.com/adafruit-tsl2591/python-circuitpython#full-example-code-2997857-15`
Perhaps the error could let the user know to lower the gain?

**Comment by jposada202020** [comment]
Confirmed the behaviour described here. tested in 

```python
Adafruit CircuitPython 6.2.0-beta.3-188-g05ed179e1 on 2021-03-15; Adafruit Feather RP2040 with rp2040
```

Problem gets solved when using `GAIN_LOW`

**Comment by jposada202020** [comment]
Solved in #21. Thanks. Closing

**Comment by averter** [comment]
The message is helpful, but my tsl2591 is under direct sunlight with `GAIN_LOW` and `INTEGRATIONTIME_100MS` and the code keeps crashing... With this configuration isn't this sensor supposed to work even under these extreme light conditions? I'm copying my code below for reference.

````python
import board
import adafruit_tsl2591
import time
import requests
i2c = board.I2C()
sensor = adafruit_tsl2591.TSL2591(i2c)
sensor.gain = adafruit_tsl2591.GAIN_LOW
sensor.integration_time = adafruit_tsl2591.INTEGRATIONTIME_100MS
READING_DELAY = 1
time.sleep(READING_DELAY)
print(sensor.lux)
````
